rdkcentral/Dobby
# RDK-45298: Checking /opt/dobby.json for settings

### Description
Added checking /opt/dobby.json for dobby settings.

### Test Procedure
Add /opt/dobby.json config file. Check in Dobby logs if it was open instead of /etc/dobby.json.

### Type of Change

### Requires Bitbake Recipe changes?
